flume-ng-dist/pom.xml

```diff
@@ -91,6 +91,29 @@
           <skip>true</skip>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>com.spotify</groupId>
+        <artifactId>dockerfile-maven-plugin</artifactId>
+        <version>${dockerfile.version}</version>
+        <!-- Wire up to the default build phases -->
```

Comment:
this plugin is in "public archived" status. we should treat it as EOL and should not rely on it.

Reply:
it's over here:
https://github.com/spotify/dockerfile-maven